src/app/DynamicImport.tsx

```diff
@@ -2,7 +2,9 @@ import * as React from 'react';
 import { accessibleRouteChangeHandler } from '@app/utils/utils';
 
 interface IDynamicImport {
+  // eslint-disable-next-line
```

Comment:
It's totally cool to disable eslint where we need to, but it might be nice to only disable specific rule(s) so it's more clear why you're disabling it for that line. You can just put the rule names separated by commas after the disable directive, like `// eslint-disable no-alert, no-console`, I think it works both with and without `-next-line`.